load-test-framework/src/main/java/com/google/pubsub/flic/controllers/mode/ModeProvider.java

```diff
@@ -0,0 +1,26 @@
+package com.google.pubsub.flic.controllers.mode;
```

Comment:
Please add the copyright header on all files.

Reply:
Done.